app/grandchallenge/forums/templates/board_base.html

```diff
@@ -12,31 +14,63 @@
 
 {% block body %}
     {% include "grandchallenge/partials/navbar.html" with hide_userlinks=False %}
-    {% include "partials/breadcrumb.html" %}
-<div class="my-5 container" id="main_container">
-    <div class="row">
-        <div class="col-12">
-            <div class="float-right controls-link-wrapper">
-                {% if not request.user.is_anonymous %}
-                    <a href="{% url 'notifications:follow-list' %}" class="d-inline-block ml-3"><i class="fas fa-bookmark">&nbsp;</i>{% trans "Subscriptions" %}</a>
-                    <a href="{% url 'forum_member:user_posts' request.user.id %}" class="d-inline-block ml-3"><i class="fas fa-comments">&nbsp;</i>{% trans "View my posts" %}</a>
-                {% endif %}
-                {% get_permission 'can_access_moderation_queue' request.user as can_access_moderation_queue %}
-                {% if can_access_moderation_queue %}
-                    <a href="{% url 'forum_moderation:queue' %}" class="d-inline-block ml-3"><i class="fas fa-gavel">&nbsp;</i>{% trans "Moderation queue" %}</a>
-                {% endif %}
+    {% if forum %}
```

Comment:
Shouldn't this be `{% if forum.challenge %}`?

Reply:
For consistency, I guess that's better, but it works like this as well because the general forum views don't have a specific `forum` in their context, so checking for `forum` is enough to tell challenge-specific forum pages from general forum pages.

Reply:
Can't say that I understand why that is, but ok!